test/cli/package-implicit-parent-namespace-export/other_package/reference.rb

```diff
@@ -0,0 +1,13 @@
+# typed: strict
+
+module OtherPackage
+  MPP = MyPackage::ParentNamespace # Allowed due to implicit parent namespace exporting
+
+  MPP::Const1
+  MPP::Const2
+  MPP::PrivateConst3 # Should be an ERROR! Private constant.
```

Comment:
If you make this into a testdata test, these "should be an error" comments never get out of date 😉

Reply:
Haha, true.